### PR TITLE
DM-39048: Add Gaussian histogram fit values to ptc dataset.

### PR DESCRIPTION
**Pull request by erykoff**

### CODE REVIEW

File: python/lsst/ip/isr/ptcDataset.py [hunk 1]
```diff
@@ -526,6 +561,15 @@ def fromTable(cls, tableList):
                     inDict['ptcTurnoff'][ampName] = np.nan
             else:
                 inDict['ptcTurnoff'][ampName] = record['PTC_TURNOFF']
+            if calibVersion == 1.1:
+                inDict['histVars'][ampName] = np.nan
```
Comment (czwa) [inline python/lsst/ip/isr/ptcDataset.py:565]: Do these need to be an array type?  `fromDict` is casting it into a numpy array above, but it won't have the same length as there are pairs of inputs (except in the partialPtc case).  Leaving this as is I think just means that any future code that wants to plot these values as a function of mean/variance needs to check the version before doing so.

Comment (czwa) [reply python/lsst/ip/isr/ptcDataset.py:565]: This also has a different default value for `kspValues` than that set in `cpPtcExtractTask.py:L535`.

**Review by czwa — APPROVED** [review]
Minor question that I'm not sure of the correct answer.

File: python/lsst/ip/isr/ptcDataset.py (hunk 1 above)
Comment (erykoff) [reply python/lsst/ip/isr/ptcDataset.py:565]: Both very good points!  I have to fix the docstring as well.

**Pull request merged**